spec/integration/alerts_spec.rb

```diff
@@ -488,20 +597,16 @@ module Hawkular::Alerts::RSpec
       events = client.list_events('thin' => true)
 
       expect(events).to_not be_nil
-      expect(events.size).to be(12)
+      expect(events.size).to be >= 12
```

Comment:
I did this so it could run on a live server, making it more tolerant to do something like VCR_UPDATE=1 bundle exec specs